### PR TITLE
Fixed error when "Search Adobe Stock" button is clicked before the grid is rendered

### PR DESCRIPTION
**Pull request by engcom-Golf**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#414: When clicked too fast on Search Adobe Stock error: this.container undefined
2. ...

### Manual testing scenarios (*)
1. Configure Adobe stock
2. Go to Catalog-> category-> content tab ->insert image-> Adobe Stock Search
3. Grid rendered correctly no errors in console.

Actual Result:
`error: container undefined` in JS console

**Before**
deepin-screen-recorder_Select area_20190904160438

**After**
deepin-screen-recorder_Select area_20190904160338

### CODE REVIEW

**Comment by sivaschenko** [comment]
Hi @engcom-Golf @Nazar65 thanks for the pull request. What do you think about adjusting the fix to wait for `this.container` being initialized in the `setLayoutStyles` function of `masonry.js` component?

Can you also sign the Adobe CLA?

**Comment by Nazar65** [comment]
Hi @sivaschenko  yes , i will try.

**Comment by engcom-Golf** [comment]
HI @sivaschenko done :heavy_check_mark:  now all works perfectly

File: AdobeUi/view/adminhtml/web/js/components/grid/masonry.js [hunk 1]
```diff
@@ -152,6 +151,28 @@ define([
             }.bind(this));
         },
 
+        /**
+         * Waits for container.
+         */
+        waitForContainer: function (callback) {
+            if (this.imports.rows.length === 0 || typeof this.container === "undefined") {
```
Comment (sivaschenko) [inline AdobeUi/view/adminhtml/web/js/components/grid/masonry.js]:
```suggestion
            if (typeof this.container === "undefined") {
```
I don't think it's necessary to check for rows length as the search result may contain 0 rows.

**Review by sivaschenko — CHANGES_REQUESTED** [review]

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**